packages/react-components/src/VoteValue.tsx

```diff
@@ -30,14 +30,19 @@ function VoteValue ({ accountId, autoFocus, isCouncil, onChange }: Props): React
   const { api } = useApi();
   const allBalances = useCall<DeriveBalancesAll>(api.derive.balances.all, [accountId]);
   const [{ selectedId, value }, setValue] = useState<ValueState>({});
+  const maxVotingBalance = useMemo(() =>
+    isCouncil
+      ? allBalances?.votingBalance.add(allBalances?.reservedBalance)
+      : allBalances?.votingBalance
+  , [allBalances?.reservedBalance, allBalances?.votingBalance, isCouncil]);
 
   useEffect((): void => {
     // if the set accountId changes and the new balances is for that id, set it
     (accountId !== selectedId) && allBalances && allBalances.accountId.eq(accountId) && setValue({
       selectedId: accountId,
-      value: allBalances.lockedBalance
+      value: allBalances?.lockedBalance
```

Comment:
```suggestion
      value: allBalances.lockedBalance
```

Checked for existence already.